src/locale/sk-SK.h

```diff
@@ -0,0 +1,393 @@
+// Copyright 2019 - David Conran (@crankyoldgit)
+// Locale/language file for Slovakia.
+// This file will override the default values located in `defaults.h`.
+#ifndef LOCALE_SK-SK_H_
+#define LOCALE_SK-SK_H_
+
+#define D_STR_UNKNOWN "NEZNÁME"
+#define D_STR_PROTOCOL "Protokol"
+#define D_STR_POWER "Výkon"
+#define D_STR_PREVIOUS "Predchádzajúce"
+#define D_STR_ON "Zap"
+#define D_STR_1 "1"
+#define D_STR_OFF "Vyp"
+#define D_STR_0 "0"
+#define D_STR_MODE "Režim"
+#define D_STR_TOGGLE "Prepnúť"
+#define D_STR_TURBO "Turbo"
+#define D_STR_SUPER "Super"
```

Comment:
When they are the same as the default, please don't include them.
i.e. You should be able to remove these two lines.